pulp_2_tests/tests/rpm/api_v2/test_errata.py

```diff
@@ -83,6 +83,7 @@ def _create_repo_file(self, cfg, repo):
                 'pulp/repos/',
                 repo['distributors'][0]['config']['relative_url']
             )),
+            name=repo['_href'],
```

Comment:
If I have a configuration file `/etc/yum.repos.d/foo.conf`, should its "name" also be "foo?" Like this:

```ini
[foo]
name = foo
enabled = 1
gpgcheck = 0
...
```

Or is it OK for "name" to be different?

Reply:
I looked into the documentation, and I did not find anything mentioning this to be mandatory.

Then I looked in my own machine. There is a repo called `fedora.repo`.
``` ini
[fedora]
name=Fedora $releasever - $basearch
...
```